Update-Dell7450Firmware.ps1

```
@@ -0,0 +1,544 @@
+        # Check for unknown variant
+        if ($drive.Variant -eq "UNKNOWN") {
+            Write-Log "Unknown variant detected: $($drive.Model)" "WARN"
+            Write-Host "`n*** Unable to determine drive variant - manual verification required ***" -ForegroundColor Yellow
+            continue
+        }
+
+        # Compare versions
+        $comparison = Compare-FirmwareVersion -Current $drive.FirmwareVersion -Target $TargetFirmwareVersion
+
+        if ($comparison -lt 0) {
+            Write-Host "Status: UPDATE REQUIRED" -ForegroundColor Yellow
```

PSScriptAnalyzer (1.24.0)

[warning] File 'Update-Dell7450Firmware.ps1' uses Write-Host. Avoid using Write-Host because it might not work in all hosts, does not work when there is no host, and (prior to PS 5.0) cannot be suppressed, captured, or redirected. Instead, use Write-Output, Write-Verbose, or Write-Information.

(PSAvoidUsingWriteHost)